kit/inputs/src/UIMaterialTextView/useMaterialTextViewChildren.tsx

```diff
@@ -84,10 +84,15 @@ export function useMaterialTextViewChildren(
     isFocused: boolean,
     isHovered: boolean,
     clear: () => void,
+    success?: boolean,
 ) {
     const clearButton = useClearButton(inputHasValue, isFocused, isHovered, clear);
-
-    if (clearButton) {
+    
+    /**
+     * If clearButton is visible it blocks displaying child icons.
+     * So we show it if the input is invalid.
```

Comment:
```suggestion
     * If clearButton is visible it blocks displaying child icons.
     * So we show it if the input is invalid.
     *
     * It was made for UIKeyTextView, as for the signing box interface in the browser
     * we want to show the icon of submitting, when a key is valid, instead of a clear button
```